vuepress/docs/next/docs/concepts/README.md

```diff
@@ -74,8 +74,8 @@ An Entando application is composed of three parts:
 
 3.  **Entando Component Manager**: the service providing the Entando
     Component Repository functionality, e.g. listing the available
-    bundles, install/uninstall a bundle, etc. Check the [dedicated
-    section](#ecm-section) for more details.
+    bundles, install/uninstall a bundle, etc. Check the dedicated Entando Component Management
```

Comment:
Did you try updating the link? I thought maybe it just changed - https://dev.entando.org/v6.3.2/docs/concepts/#entando-component-repository instead of #ecm.

Reply:
am I missing something. Where is the "ecm section" header?

Reply:
Heh, this page needs to be rewritten. Let's just drop the sentence altogether since there's not enough content in the later sections to be worth a link.

Reply:
i'll strike the sentence, but let's discuss in today's meeting...